project-koku/koku
# COST-1293 changed description filter method to allow more than one filter

Fixes [COST-1293](https://issues.redhat.com/browse/COST-1293)

Changes proposed in this PR:
* Changed cost-model 'description' filter method to the same as 'name' field to allow more than one filter 

Testing Instructions:
1. Send a request to the following endpoint with 2 or more filters for 'description' field:
   - http://127.0.0.1:8000/api/cost-management/v1/cost-models/?limit=10&offset=0&description=model,aws
2. Verify this request returns data
